Services/AuthenticationService/Validators/UserEmailPasswordValidator.cs

```diff
@@ -13,11 +13,11 @@ public UserEmailPasswordValidator()
                 .EmailAddress()
                 .WithMessage("Email address must be in valid format.");
 
-            RuleFor(user => user.Password)
+            RuleFor(user => user.PasswordHash)
                 .NotEmpty()
-                .WithMessage("Password must not be empty.")
-                .Length(5, 50)
-                .WithMessage("Password's length must be from 5 to 50 symbols.");
+                .WithMessage("Password hash must not be empty.")
+                .Length(40)
+                .WithMessage("Password's hash length must be 40 symbols.");
```

Comment:
Exactly 40 each time? How do we get hash btw?

Reply:
We discussed the option with SHA